README.md

```diff
@@ -1,2 +1,115 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
+
+## Getting Started
+
+- Clone a copy of the master branch in the TAU Git repository:
+
+`git clone git@github.com:Samsung/TAU.git`
+
+- Change to the tau directory:
+
+`cd TAU`
+
+- Check out the latest version of TAU:
+
+`git checkout tizen_X.Y`
+
+eg. `git checkout tizen_5.0`
+
+### Prerequisites
+
+To build a TAU library, ensure that you have both Git and Node.js (6.16.0) installed.
+
+Example install using `nvm`:
+
+```
+nvm install 6.16.0
+nvm alias default 6.16.0
+```
+
+- Test that you have grunt installed:
+
+`grunt -V`
+
+If grunt is not installed, then run:
+
+`(sudo) npm install -g grunt-cli`
+
+### Installing
+
+- Install the build module by npm:
+
+`npm install`
+
+Build TAU by running the following command in the tau directory:
+
+`grunt build`
+
+## Running the tests
+
+TAU has automated tests. For run tests execute command:
+
+`grunt test`
+
+## Demo application:
+
+UIComponents sample web application demonstrating how to implement UI Components for wearable based on TAU.
+
+Path in repository:
+
+mobile: `examples\mobile\UIComponents`
+
+wearable: `examples\wearable\UIComponents`
+
+- Install http-server
+
+`npm install http-server -g`
+
+- Launch server in app directory:
+
+`http-server -p <port> -a localhost`
+
+eg. `http-server -p 8888 -a localhost`
+
+
+### And coding style tests
+
+Project has defined eslint rules (.eslintrc.js)
+
+Contribution can be verified using command:
+
+`grunt eslint`
+
+## Deployment
+
+@todo
+
+## Built With
+
+@todo
+
+## Contributing
+
+@todo
+Please read [CONTRIBUTING.md] for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+@todo
+
+## Authors
+
+* Authors - [AUTHOR]
+
+See also the list of [contributors](https://github.com/SAMSUNG/TAU/contributors) who participated in this project.
+
+## License
+
```

Comment:
this section duplicates function of COPYING file which states 
This software is licensed under the MIT license (as defined
by the OSI at opensource.org/licenses/MIT), except for
following files:...
How about refering to this file and stating that 

this project is licensed under MIT except some files under Flora license. For details see COPYING...

Reply:
Sounds better... thank you.